srcpkgs/mesa/template

```diff
@@ -337,7 +331,7 @@ mesa-vulkan-overlay-layer_package() {
 mesa-ati-dri_package() {
 	build_style=meta
 	lib32mode=full
-	short_desc="Mesa DRI drivers for ATI GPUs (transitional dummy package)"
+	short_desc="Mesa DRI drivers for ATI GPUs"
```

Comment:
If you want to make this a legit metapackage, I'd change `short_desc` to a more generic "Mesa drivers for ATI GPUs", and same for Intel and Nouveau

Reply:
let's not make it legit metapackages

they are still transitional dummy packages that people should not install by default, they should just remain there since they cause no harm and removing them would break upgrading old systems

Reply:
Thing is that they install packages other than only `mesa-dri`... They provide a certain convenience factor, imo.